feat(calendar): add accessibility

goremikins opened the pull request:
## Description
Improves screen reader support

## Type of change
- [X] New feature (non-breaking change which adds functionality)

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

wsuwt commented on packages/elements/src/calendar/index.ts:
```diff
@@ -378,16 +455,44 @@ export class Calendar extends ControlElement implements MultiValue {
+    // This code is here to ensure that focus is not lost
+    // while navigating through the render views using keyboard
+    if (this.focused && changedProperties.has('renderView') && this.activeElement !== this.viewBtnRef.value) {
+      this.viewBtnRef.value!.focus();
```
I know view button is always there, but asserting null throws a warning. Should we use `?` instead to only focus when button is really there ?

goremikins replied:
Done

Jidapa-Pai commented:
I'm already checked issue about tap on year after merge this code https://github.com/Refinitiv/refinitiv-ui/pull/81 from develop, it works now.

goremikins commented:
@TremayneChrist , @wsuwt , I think this need another walkthrough. Most notably I have changed the following:
1.  IE11, navigation did not work
2. IE11, it took absolutely ages to load the calendar. The reason is because polyfills are required to render formatted dates. So I removed `aria-label` for all elements that cause problem in IE11
3. Simplified some types
4. Changed the way how navigation maps are calculated to keep the code cleaner
